### ISSUE TITLE
Quickstart Colab notebooks not working

### ISSUE THREAD

**Issue by kbschliep** [webis-de/small-text#19]
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-13-6d9e5b75164c>](https://localhost:8080/#) in <module>
      2 
      3 
----> 4 train = TransformersDataset.from_arrays(raw_dataset['train']['text'],
      5                                         raw_dataset['train']['label'],
      6                                         tokenizer,

AttributeError: type object 'TransformersDataset' has no attribute 'from_arrays'

**Comment by chschroeder** [comment]
TL;DR: The problem has been fixed for the main branch. The current docs still have that problem until I make a bugfix release `v1.0.1` or finalize the next version `v.1.1.0`.

The notebook links pointed to the main branch in which the notebook code was adapted to reflect recent code changes except for the version which still pointed at `1.0.0`. The ideal behavior is probably (besides reflecting about my git branching workflow) that the README.md / docs always point to the latest release. I have fixed this for the main branch.

Still, notebooks examples that are not working is an awkward situation. Thank you very much for reporting that issue, @kbschliep!

**Comment by chschroeder** [comment]
I just released `v1.0.1`. Everything should be fixed now :+1:.
